### PR TITLE
Home user interface done

### PR DESCRIPTION
**Pull request by ShivaaniDushya**

### CODE REVIEW

**Review by amanthajayathilake — CHANGES_REQUESTED** [review]
In App.js
**Remove this and**
import { MainStackNavigator } from './src/navigation/stackNavigator.js';

**Add**
import { MainStackNavigator } from './src/navigation/StackNavigator.js';

**Review by Shehanx86 — APPROVED** [review]

**Review by dulshan-perera — APPROVED** [review]
LGTM

File: src/pages/UserProfile/UpdateProfile.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+                        value={value}
+                        items={items}
+                        setOpen={setOpen}
+                        setValue={setValue}
+                        setItems={setItems}
+                        style={userProfileStyles.inputContainerUpdateProfile}
+                        placeholderStyle={{
+                            color: "grey",
+                        }}
+                    />
```
Comment (dulshan-perera) [inline src/pages/UserProfile/UpdateProfile.js:45]: There is a reusable dropdown component in the Components section.

**Review by dulshan-perera — CHANGES_REQUESTED** [review]
Check the reusable dropdown component

**Review by amanthajayathilake — APPROVED** [review]

**Review by dulshan-perera — APPROVED** [review]
Changes done. Minor issues exist. Let's fix them in the next update.

**Pull request merged**